python_modules/automation/automation/docker/images/buildkite-test/Dockerfile

```diff
@@ -97,6 +96,15 @@ RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - \
     # Confirm Java works
     && java -version
 
+# Install FOSSA cli for analyzing our open-source dependencies
+# https://github.com/fossas/fossa-cli
+RUN wget https://github.com/fossas/fossa-cli/releases/download/v${FOSSA_VERSION}/fossa-cli_${FOSSA_VERSION}_linux_amd64.tar.gz \
+   && tar xvzf fossa-cli_${FOSSA_VERSION}_linux_amd64.tar.gz fossa \
+   && chmod +x ./fossa \
+   && mv ./fossa /usr/local/bin/fossa \
+   && rm fossa-cli_${FOSSA_VERSION}_linux_amd64.tar.gz \
+   && fossa --version
```

Comment:
Why not use their recommended method of the [v1 install script](https://github.com/fossas/fossa-cli/blob/master/docs/walkthroughs/installing-fossa-cli.md#installing-cli-1x-using-installation-script)?